tests/pine.spec.coffee

```diff
@@ -37,7 +40,7 @@ describe 'Pine:', ->
 
 		it "should equal /#{apiVersion}/", ->
 			pine = buildPineInstance(mockServer.url)
-			m.chai.expect(pine.apiPrefix).to.equal(pine.API_PREFIX)
+			m.chai.expect(pine.apiPrefix).to.equal(pine.apiPrefix)
```

Comment:
This isn't the most useful test anymore :P

Reply:
I just realized what it does... :joy: